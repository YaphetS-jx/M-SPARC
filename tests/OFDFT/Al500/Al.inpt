# Test: Al #
CELL: 40 40 40
MESH_SPACING: 0.5
FD_ORDER: 6
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ

TOL_POISSON: 1.00E-08
TOL_PSEUDOCHARGE: 1.00E-08

OFDFT_FLAG: 1
TOL_OFDFT: 1E-3
OFDFT_LAMBDA: 0.2