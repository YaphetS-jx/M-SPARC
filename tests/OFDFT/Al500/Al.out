***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 24-Mar-2021  Start time: 01:23:54                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 40.000000 40.000000 40.000000 
FD_GRID: 80 80 80
FD_ORDER: 6
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
EXCHANGE_CORRELATION: LDA_PZ
OFDFT_FLAG: 1
TOL_OFDFT: 1.00E-06
CALC_STRESS: 0
CALC_PRES: 0
TOL_PSEUDOCHARGE: 1.00E-08
REFERENCE_CUTOFF: 0.50
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/OFDFT/Al500/Al.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.500000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/OFDFT/Al500/Al.out
Total number of atom types         :  1
Total number of atoms              :  500
Total number of electrons          :  1500
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  Al_goodwin.psp8
lloc                               :  0
Pseudocharge radii of atom type 1  :  10.00 10.00 10.00
Number of atoms of type 1          :  500
Estimated total memory usage       :  14.57 GB
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#1)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
0              -2.2117957040E+00       1.064E-02         2.372
1              -2.2126455819E+00       6.804E-03         1.748
2              -2.2132128588E+00       8.746E-03         1.865
3              -2.2130923737E+00       2.625E-03         1.888
4              -2.2132219476E+00       1.851E-03         1.634
5              -2.2132692722E+00       2.911E-03         1.631
6              -2.2132505774E+00       1.208E-03         1.862
7              -2.2132668144E+00       4.703E-04         1.626
8              -2.2132714609E+00       8.284E-04         1.630
9              -2.2132706602E+00       2.808E-04         1.626
10             -2.2132716856E+00       9.364E-05         1.630
11             -2.2132719545E+00       2.151E-04         1.631
12             -2.2132718887E+00       8.331E-05         1.861
13             -2.2132719681E+00       2.288E-05         1.625
14             -2.2132719885E+00       5.712E-05         1.630
15             -2.2132719843E+00       2.099E-05         1.863
16             -2.2132719895E+00       4.968E-06         1.696
17             -2.2132719898E+00       1.098E-05         1.633
18             -2.2132719902E+00       1.727E-05         1.629
19             -2.2132719899E+00       6.146E-06         2.322
20             -2.2132719900E+00       2.954E-06         2.097
21             -2.2132719903E+00       3.003E-05         1.861
22             -2.2132719893E+00       2.121E-05         2.786
23             -2.2132719903E+00       1.166E-06         2.089
24             -2.2132719903E+00       1.197E-06         1.860
25             -2.2132719903E+00       1.794E-06         1.629
26             -2.2132719903E+00       4.346E-06         1.628
27             -2.2132719903E+00       2.413E-06         2.554
28             -2.2132719903E+00       5.287E-07         2.286
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2132719903E+00 (Ha/atom)
Total free energy                  : -1.1066359952E+03 (Ha)
Kinetic energy                     :  4.1305482119E+02 (Ha)
Exchange correlation energy        : -3.9320663094E+02 (Ha)
Self and correction energy         : -2.0477261141E+03 (Ha)
Average force                      :  1.8201797148E-07 (Ha/Bohr)
Maximum force                      :  2.6115899698E-07 (Ha/Bohr)
Time for force calculation         :  19.092 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  91.687 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
